herddb-core/src/main/java/herddb/core/TableManager.java

```diff
@@ -389,6 +389,18 @@ public Table getTable() {
                     return table;
                 }
             };
+        } else if (table.getColumn(table.primaryKey[0]).type == ColumnTypes.NOTNULL_INTEGER) {
```

Comment:
can you merge this branch with the ColumnTypes.INTEGER one ?

Reply:
Done.